modal/volume.py

```diff
@@ -499,7 +513,14 @@ async def batch_upload(self, force: bool = False) -> "_VolumeUploadContextManage
             batch.put_file(io.BytesIO(b"some data"), "/foobar")
         ```
         """
-        return _VolumeUploadContextManager(self.object_id, self._client, force=force)
+        from modal_proto.api_pb2 import VolumeFsVersion
```

Comment:
This import can probably be global?

Reply:
I didn't see an existing pattern of importing proto messages into the global scope, so I wanted to limit the blast radius. I'll happily hoist it up if you think it is a good idea.

Reply:
(Please confirm if you still want me to do that given my reply above)

Reply:
The established pattern is to do `from modal_proto import api_pb2` and then reference the proto messages using `api_pb2.X`

Reply:
Yeah, I used to have that before but felt like the line got too long, hence the import. But I can move back to it, sounds good!